docs/v6/configuration/optional-configuration.md

```diff
@@ -54,6 +54,78 @@ configuring them in unison.
         START_LOCAL_UNIVERSE = False
         START_SCHEDULER_UNIVERSE = $(START_LOCAL_UNIVERSE)
 
+Inserting IDTOKENs into the routed job's sandbox
+------------------------------------------
+
+If you want to insert IDTOKENS into the routed job's sandbox you can use the `SendIDTokens` route command, or
+the `JOB_ROUTER_SEND_ROUTE_IDTOKENS` global configuration variable. Tokens
+sent using this mechanism must be named and declared using the `JOB_ROUTER_CREATE_IDTOKEN_NAMES`
+and [`JOB_ROUTER_CREATE_IDTOKEN_<name>`](https://htcondor.readthedocs.io/en/latest/admin-manual/configuration-macros.html#JOB_ROUTER_CREATE_IDTOKEN_%3CNAME%3E) configuration variables.  Tokens whose names are declared in
+the `JOB_ROUTER_SEND_ROUTE_IDTOKENS` configuration variable are sent by default for each route that does
+not have a `SendIDTokens` command.
+
+- **To declare IDTOKENS for inclusion in glide-in jobs** for the purpose of advertising to a collector
+  add something like the following to `/etc/condor-ce/config.d/99-local-ce-token.conf`:
+
+        JOB_ROUTER_CREATE_IDTOKEN_NAMES = name1 name2
+        JOB_ROUTER_CREATE_IDTOKEN_name1 @=end
+            sub = "name1_pilot@domain"
+            kid = "POOL"
+            lifetime = 3900
+            scope = "ADVERTISE_STARTD, ADVERTISE_MASTER, READ"
+            dir = "/etc/condor-ce/gltokens/name1"
+            filename = "ce_name1.idtoken"
+            owner = "owner1"
+        @end
+        JOB_ROUTER_CREATE_IDTOKEN_Name2 @=end
+            sub = "name2@users.htcondor.org"
+            kid = "POOL"
+            lifetime = 3900
+            scope = "ADVERTISE_STARTD, ADVERTISE_MASTER, READ"
+            dir = "/etc/condor-ce/gltokens/name2"
+            filename = "ce_name2.idtoken"
+            owner = "owner2"
+        @end
+
+- **To insert one of the above IDTOKENS in the sandbox of a routed job**, include the token name in the `SendIDTokens` route
+   command like this.
+
+        SendIDTokens = "Name2"
+
```

Comment:
```suggestion
    !!! note "Route commands"
        `SendIDTokens` is a route command, not a job attribute.
        This means that you will not be able to manipulate it through
        [transform verbs](writing-job-routes.md#editing-attributes) such as `EVALSET`.
```

I just realized that my question to you yesterday about using `EVALSET` in a pre/post route transform may be a common one